store/src/java/com/zimbra/qa/unittest/TestRemoteImapShared.java

```diff
@@ -37,38 +42,42 @@ public void tearDown() throws ServiceException, DocumentException, ConfigExcepti
     }
 
     @Override
-    @Ignore ("failing on remote imap for now")
-    public void testMultiappendNoLiteralPlus() throws Exception {
-
+    protected ZMailbox getImapZMailbox() throws Exception {
```

Comment:
Please add a comment on why this exists.  We may want to deprecate it at some point in the future if we get to a point where all ZMailboxes are in sync.